src/eth/GasEstimatorService.js

```diff
@@ -3,62 +3,88 @@ import { PublicService } from '@makerdao/services-core';
 export default class GasEstimatorService extends PublicService {
   constructor(name = 'gasEstimator') {
     super(name, ['web3', 'log']);
-    this._percentage = null;
+    this._multiplier = 1.55;
     this._absolute = null;
+    this._fallback = 4000000;
   }
 
-  estimateGasLimit(transaction) {
-    if (this._percentage === null && this._absolute === null) {
-      throw new Error('no gas limit policy set');
+  async estimateGasLimit(transaction, options = {}) {
```

Comment:
it would be better here just to return the bare value rather than mutating options